aim/web/api/runs/pydantic_models.py

```diff
@@ -140,7 +140,7 @@ class StructuredRunUpdateIn(BaseModel):
 
 
 class StructuredRunsArchivedIn(BaseModel):
```

Comment:
looks like this class is not used anymore. please remove.

Reply:
Ok, @alberttorosyan jan.